java/src/main/java/io/crate/jdbc/sample/Controller.java

```diff
@@ -27,14 +27,30 @@
     private static final int CREATED = 201;
     private static final int OK = 200;
 
-    public Controller(final DataProvider model) {
+    Controller(final DataProvider model) {
 
         before(((request, response) -> {
             response.header("Access-Control-Allow-Origin", "*");
             response.header("Access-Control-Request-Method", "*");
+            response.header("Access-Control-Allow-Methods", "*");
             response.header("Access-Control-Allow-Headers", "*");
         }));
 
+        options("/*", (request, response) -> {
```

Comment:
don't quite understand this behaviour.
why are certain headers checked for existence, others not?